manifest.json

```diff
@@ -0,0 +1,132 @@
+{
+    "app-id": "org.kiwix.Client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "kiwix-desktop",
+    "rename-desktop-file": "kiwix-desktop.desktop",
```

Comment:
Ideally these should get renamed in upstream repo.

Reply:
I'm not sure about this.
The project is named `kiwix-desktop` and it can be installed another way that with flathub.
(We create appimage already and may create "classic" packages)
I prefer to keep a generic name in upstream repository and have flatpak specific name in the flatpak side.

Reply:
It's mentioned in desktop launcher and appdata specifications that appid form is recommended. It's not flatpak specific.

Reply:
But we use appid only in flatpak. Everywhere else, the name is `kiwix-desktop`.

Reply:
We don't rename it anymore. We use the correct "org.kiwix.desktop" name in the upstream project.